<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TPFourClass</class>
 <widget class="QMainWindow" name="TPFourClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TPFour</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="puissance">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>19</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature</string>
    </property>
   </widget>
   <widget class="QLabel" name="valuePuissance">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>130</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Valeur : 0%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fourOn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Allumer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fourOff">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Eteindre</string>
    </property>
   </widget>
   <widget class="QListWidget" name="affichage">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TPFour.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fourOn</sender>
   <signal>clicked()</signal>
   <receiver>TPFourClass</receiver>
   <slot>fourOn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fourOff</sender>
   <signal>clicked()</signal>
   <receiver>TPFourClass</receiver>
   <slot>fourOff()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>puissance</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TPFourClass</receiver>
   <slot>updateUI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fourOn()</slot>
  <slot>fourOff()</slot>
  <slot>updateUI()</slot>
 </slots>
</ui>
